features/git-town-new-pull-request/bitbucket.feature

```diff
@@ -11,22 +11,23 @@ Feature: git-new-pull-request when origin is on Bitbucket
+      | https://username@bitbucket.org/Originate/git-town                 | https://bitbucket.org/Originate/git-town/pull-request/new?dest=Originate%2Fgit-town%3A%3Amain&source=Originate%2Fgit-town%.*%3Afeature                                     |
+      | git@bitbucket.org/Originate/git-town.git                          | https://bitbucket.org/Originate/git-town/pull-request/new?dest=Originate%2Fgit-town%3A%3Amain&source=Originate%2Fgit-town%.*%3Afeature                                     |
+      | git@bitbucket.org/Originate/git-town                              | https://bitbucket.org/Originate/git-town/pull-request/new?dest=Originate%2Fgit-town%3A%3Amain&source=Originate%2Fgit-town%.*%3Afeature                                     |
+      | git@bitbucket-as-account2:Originate/git-town.git                  | https://bitbucket.org/Originate/git-town/pull-request/new?dest=Originate%2Fgit-town%3A%3Amain&source=Originate%2Fgit-town%.*%3Afeature                                     |
+      | http://username@bitbucket.org/Originate/originate.github.com.git  | https://bitbucket.org/Originate/originate.github.com/pull-request/new?dest=Originate%2Foriginate.github.com%3A%3Amain&source=Originate%2Foriginate.github.com%.*%3Afeature |
+      | http://username@bitbucket.org/Originate/originate.github.com      | https://bitbucket.org/Originate/originate.github.com/pull-request/new?dest=Originate%2Foriginate.github.com%3A%3Amain&source=Originate%2Foriginate.github.com%.*%3Afeature |
+      | https://username@bitbucket.org/Originate/originate.github.com.git | https://bitbucket.org/Originate/originate.github.com/pull-request/new?dest=Originate%2Foriginate.github.com%3A%3Amain&source=Originate%2Foriginate.github.com%.*%3Afeature |
+      | https://username@bitbucket.org/Originate/originate.github.com     | https://bitbucket.org/Originate/originate.github.com/pull-request/new?dest=Originate%2Foriginate.github.com%3A%3Amain&source=Originate%2Foriginate.github.com%.*%3Afeature |
+      | git@bitbucket.org/Originate/originate.github.com.git              | https://bitbucket.org/Originate/originate.github.com/pull-request/new?dest=Originate%2Foriginate.github.com%3A%3Amain&source=Originate%2Foriginate.github.com%.*%3Afeature |
+      | git@bitbucket.org/Originate/originate.github.com                  | https://bitbucket.org/Originate/originate.github.com/pull-request/new?dest=Originate%2Foriginate.github.com%3A%3Amain&source=Originate%2Foriginate.github.com%.*%3Afeature |
+      | ssh://git@bitbucket.org/Originate/git-town.git                    | https://bitbucket.org/Originate/git-town/pull-request/new?dest=Originate%2Fgit-town%3A%3Amain&source=Originate%2Fgit-town%.*%3Afeature                                     |
+      | ssh://git@bitbucket.org/Originate/git-town                        | https://bitbucket.org/Originate/git-town/pull-request/new?dest=Originate%2Fgit-town%3A%3Amain&source=Originate%2Fgit-town%.*%3Afeature                                     |
```

Comment:
This is so verbose! I think we can move a lot of this to unit tests where we get the hostname from the origin url. Then we could get away with only two examples here: one where the host is bitbucket, and the other when the host is an ssh identity: bitbucket-as-account2

Reply:
I have started a spike of a unit test. It ends up even more verbose. We would have to use a [table-driven test](https://github.com/golang/go/wiki/TableDrivenTests), and the table that maps origin to URL would be harder to read than the Gherkin table here. Plus all the boilerplate for the unit test stuff. I believe keeping all examples of PR URLs here in Gherkin is the cleanest, simplest, and most developer-friendly way to test the actual behavior of the tool.